README.rst

```diff
@@ -17,30 +17,97 @@
 
 .. index-start-marker
 
-Installer for D-Wave's Ocean Tools.
+`Ocean <https://docs.ocean.dwavesys.com/en/stable>`_ is
+`D-Wave's <https://www.dwavesys.com>`_ suite of tools for solving hard problems
+with quantum computers.
 
+.. index-end-marker
 
 Installation
-------------
+============
 
 .. installation-start-marker
 
-Compatible with Python 3.6+:
+Installation from `PyPI <https://pypi.org/project/dwave-ocean-sdk/>`_:
 
 .. code-block:: bash
 
     pip install dwave-ocean-sdk
 
-To install from source:
+For more information, see the Ocean documentation's
+`installation <https://docs.ocean.dwavesys.com/en/stable/overview/install.html>`_
+page.
 
-.. code-block:: bash
+.. installation-end-marker
 
-    python setup.py install
+Getting Started
+===============
 
-.. installation-end-marker
+Sign up for the Leap quantum cloud service here:
+`Leap signup <https://cloud.dwavesys.com/leap/signup>`_. Leap provides a
+browser-based integrated development environment (IDE) with Ocean preinstalled.
+
+Start learning with the following D-Wave resources:
+
+* `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_ to
+  learn about quantum computers and how to use them.
+
+* `Getting Started with Ocean <https://docs.ocean.dwavesys.com/en/stable/getting_started>`_
+  to install and start coding with Ocean software.
+
+* `dwave-examples <https://github.com/dwave-examples>`_ for code examples
+  and Jupyter Notebooks.
+
+* `D-Wave website <https://www.dwavesys.com/learn/resource-library>`_ for
+  whitepapers and additional resources.
+
+Example Quantum Program
+-----------------------
+
+The following lines of code solve and visualize a
+`random <https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generators.html>`_
+problem on a quantum computer.
+
+.. code-block:: python
+
+  import dimod
+  import dwave.inspector
+  import dwave.system
+
+  bqm = dimod.generators.ran_r(1, 20)
+  sampler = dwave.system.EmbeddingComposite(dwave.system.DWaveSampler())
+  sampleset = sampler.sample(bqm, num_reads=100)
+  dwave.inspector.show(sampleset)
+
+The left side of the visualized solution represents the problem's variables as
+circles, with white dots for variables assigned values of -1 and blue dots for
+values of +1; the colors of the connecting lines represent values of the
+quadratic coefficients for each pair of variables. The right side shows the qubits
+representing these variables on a quantum processing unit.
```

Comment:
I feel this whole explanation will be lost on the uninitiated reader. Perhaps we shouldn't even bother explaining these details here, but instead point interested users to the [inspector intro docs](https://docs.ocean.dwavesys.com/projects/inspector/en/latest/intro.html), or similar?

Reply:
OTOH, without this paragraph, the example section would be pretty bare bones (code + image), so probably better to just leave it in.

Reply:
I turned the `visualized` into a link to https://docs.ocean.dwavesys.com/en/stable/docs_inspector/intro.html so the uninitiated interested in initiation can easily access more info

Reply:
How about we also link "problem" in that sentence with https://docs.ocean.dwavesys.com/en/stable/concepts/bqm.html#bqm-sdk?

Reply:
I would love if we could link from code block -- so `DWaveSampler`, `EmbeddingComposite`, etc..

Reply:
I linked problem to `https://docs.ocean.dwavesys.com/en/stable/concepts/bqm.html`
My guess is that users will look at this to get an overall impression of what problem-submission code looks like, this is not likely to be a launching point for getting more deeply acquainted with Ocean code entities.